src/engine/SCons/Tool/packaging/__init__.py

```diff
@@ -117,8 +117,21 @@ def Package(env, target=None, source=None, **kw):
     # load the needed packagers.
     def load_packager(type):
         try:
-            file,path,desc=imp.find_module(type, __path__)
-            return imp.load_module(type, file, path, desc)
+            if PY2:
+                import imp
+                file,path,desc=imp.find_module(type, __path__)
+                return imp.load_module(type, file, path, desc)
+            else:
+                import importlib.util, importlib.machinery
```

Comment:
This all seems complicated... why not:
```
    savepath = sys.path
    sys.path = __path__
    mod = importlib.import_module(type)
    sys.path = savepath
```

Reply:
Do you have similarly brilliant way how to rewrite ``src.engine.SCons.Script.Main._load_site_scons_dir`` ? I have tried [to do it](https://github.com/mcepl/scons/blob/replace-imp-with-importlib/src/engine/SCons/Script/Main.py#L765), but it still doesn't seem to make much sense.

Reply:
Applied.